Sources/Storage/Disk/DiskStorage.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  DiskStorage.swift
+//  StatsdClient-iOS
+//
+//  Created by Nghia Tran on 10/8/17.
+//  Copyright © 2017 StatsdClient. All rights reserved.
+//
+
+import Foundation
+
+final class DiskStorage<Element: Codable>: Storage {
+
+    typealias Key = String
```

Comment:
remove all `typealias` and let Swift infer the type based on the functions signature